tests/integration/integration/login/login-hi.spec.js

```diff
@@ -0,0 +1,39 @@
+import { Selector } from "testcafe";
```

Comment:
does this mean we have to copy-paste every test for testing different languages? seems like people would have figured out a better solution for this, no?

if that is the case, let's not make the hindi/english distinction now, and let's keep everything in English.

Reply:
To have it, I've seen recommendations on using the locale JSON and having a loop for every language. We can try that if you prefer. I'll open up a separate task.

Will delete the `hindi` spec for now and rename

Reply:
Created #601